### ISSUE TITLE
Not working with Xcode 10.0 and Swift 4.2

### ISSUE THREAD

**Issue by NeverwinterMoon** [RxSwiftCommunity/RxNimble#33]
RxNimble 4.1.0
Nimble 7.1.3

I am getting
```Cycle in dependencies between targets 'RxNimble' and 'Nimble'; building could produce unreliable results. This usually can be resolved by moving the target's Headers build phase before Compile Sources.
Cycle path: RxNimble → Nimble → RxNimble
Cycle details:
→ Target 'RxNimble' has target dependency on Target 'Nimble'
...
```

**Comment by cruisediary** [comment]
#34 merged to support Xcode 10 with swift 4.2 :)

**Comment by cruisediary** [comment]
I will close this issue :)
